code/game/atoms.dm

```diff
@@ -231,7 +231,7 @@ its easier to just keep the beam vertical.
 /atom/proc/examine(mob/user)
 	//This reformat names to get a/an properly working on item descriptions when they are bloody
 	var/f_name = "\a [src]."
-	if(src.blood_DNA && !istype(src, /obj/effect/decal))
+	if(src.blood_DNA.len && !istype(src, /obj/effect/decal))

```

Comment:
I think you need to do `blood_DNA && blood_DNA.len && ...` , blood_DNA can be null afaik.